### PR TITLE
Fix linting errors

### PR DESCRIPTION
**Pull request by tijmenb**
In https://github.com/alphagov/gds-api-adapters/pull/628 I'm attempting a large scale refactor. The linting won't pass for that PR because there are styleguide violations in all the files it's touching (we normally only run rubocop against changed files).

This fixes all styleguide violations so that that PR can be merged without noise.

### CODE REVIEW

File: jenkins.sh [hunk 1]
```diff
@@ -47,8 +47,6 @@ for version in 2.3 2.2 2.1; do
   if [[ ${GIT_BRANCH} != "origin/master" ]]; then
     echo "Running ruby linter for $version"
     bundle exec govuk-lint-ruby \
-      --diff \
-      --cached \
```
Comment (danielroseman) [inline jenkins.sh]: Did you mean to delete these?

File: lib/gds_api/test_helpers/business_support_helper.rb [hunk 2]
```diff
@@ -15,28 +15,27 @@ def setup_business_support_stubs(endpoint, path)
...
 
-      def api_has_business_support(business_support, facets={})
+      def api_has_business_support(business_support, facets = {})
         facets = sanitise_facets(facets)
-        if business_support.is_a?(Symbol)
-          bs_with_facets = facets.merge(:title => business_support)
-        else
-          bs_with_facets = facets.merge(business_support)
-        end
+        bs_with_facets = if business_support.is_a?(Symbol)
```
Comment (danielroseman) [inline lib/gds_api/test_helpers/business_support_helper.rb]: We've previously discussed disabling the Style/ConditionalAssignment cop that enforces this; it leads to very ugly results here and in a few cases below.

File: lib/gds_api/test_helpers/common_responses.rb [hunk 3]
```diff
@@ -27,15 +27,13 @@ def response_base
...
-            "description" => "Tags!",
-            "total" => 100,
-            "start_index" => 1,
-            "page_size" => 100,
-            "current_page" => 1,
-            "pages" => 1,
-            "results" => []
-          }
+          "description" => "Tags!",
+"total" => 100,
```
Comment (danielroseman) [inline lib/gds_api/test_helpers/common_responses.rb]: indentation

File: lib/gds_api/test_helpers/email_alert_api.rb [hunk 4]
```diff
@@ -62,19 +62,17 @@ def stub_any_email_alert_api_call
...
-        if attributes
-          matcher = ->(request) do
-            payload = JSON.parse(request.body)
-            payload.select { |k, _| attributes.key?(k) } == attributes
-          end
-        else
-          matcher = nil
-        end
+        matcher = if attributes
+                    ->(request) do
```
Comment (danielroseman) [inline lib/gds_api/test_helpers/email_alert_api.rb]: Another case where Style/ConditionalAssignment makes this unreadable.

File: test/list_response_test.rb [hunk 5]
```diff
@@ -2,24 +2,22 @@
...
 
     it "should allow Enumerable access to the results array" do
       data = {
-        "results" => [
-          "foo",
-          "bar",
-          "baz",
-        ],
+        "results" => %w(
+foo
```
Comment (danielroseman) [inline test/list_response_test.rb]: Should probably be all on one line

File: lib/gds_api/test_helpers/common_responses.rb (hunk 3 above)
Comment (tijmenb) [inline lib/gds_api/test_helpers/common_responses.rb]: That's a really weird thing for rubocop to get wrong :|

File: lib/gds_api/test_helpers/content_api/artefact_stub.rb [hunk 6]
```diff
@@ -38,19 +38,20 @@ def stub
...
-          end
 
-          # Ensure that all keys and values are strings 
+      private
+
+        def url_without_query
+          "#{CONTENT_API_ENDPOINT}/#{CGI.escape(slug)}.json"
+        end
+
+          # Ensure that all keys and values are strings
```
Comment (MatMoore) [inline lib/gds_api/test_helpers/content_api/artefact_stub.rb]: It might be the way github is showing the diff, but it looks like this comment is overindented now.

File: lib/gds_api/test_helpers/business_support_helper.rb (hunk 2 above)
Comment (tijmenb) [inline lib/gds_api/test_helpers/business_support_helper.rb]: This isn't that cop actually. I get the following report:

```
lib/gds_api/test_helpers/business_support_helper.rb:25:11: C: Style/IndentationWidth: Use 2 (not -15) spaces for indentation.  (https://github.com/bbatsov/ruby-style-guide#spaces-indentation)
          facets.merge(title: business_support)
          ^^^^^^^^^^^^^^^
```

I think the `if`, `else` and `end` should be aligned. I'll check if I can make the linter pass differently.

File: jenkins.sh (hunk 1 above)
Comment (tijmenb) [inline jenkins.sh]: Yeah, but I should've done that in a separate commit. Removed for now. Thanks!

File: lib/gds_api/test_helpers/business_support_helper.rb (hunk 2 above)
Comment (tijmenb) [inline lib/gds_api/test_helpers/business_support_helper.rb]: Ok, I think I got it. The style used here doesn't pass the indentation rules, but `Style/ConditionalAssignment` doesn't allow us to use 

```
if bla
 foo = "a"
else
 foo = "b"
end
```

Seems like cops fighting for jurisdiction 👮. I've disabled `Style/ConditionalAssignment` and changed the examples to the "double assignment" to resolve this.

File: lib/gds_api/test_helpers/common_responses.rb (hunk 3 above)
Comment (tijmenb) [inline lib/gds_api/test_helpers/common_responses.rb]: Fixed.

File: lib/gds_api/test_helpers/email_alert_api.rb (hunk 4 above)
Comment (tijmenb) [inline lib/gds_api/test_helpers/email_alert_api.rb]: Fixed.

File: lib/gds_api/test_helpers/content_api/artefact_stub.rb (hunk 6 above)
Comment (tijmenb) [inline lib/gds_api/test_helpers/content_api/artefact_stub.rb]: Yep, correct. Fixed!

**Comment by tijmenb** [comment]
Thanks for the review @danielroseman @MatMoore - fixed the issues!
